apps/src/lib/ui/accounts/DeleteAccountDialog.story.jsx

```diff
@@ -35,16 +36,61 @@ export default storybook => {
         )
       },
       {
-        name: 'Delete Teacher Account',
-        description: 'Warning message for teacher account deletion',
+        name: 'Delete Teacher Account - without students',
+        description: 'Warning message for teacher account deletion.',
+        story: () => (
+          <DeleteAccountDialog
+            {...DEFAULT_PROPS}
+            isTeacher={true}
+            warnAboutDeletingStudents={false}
+          />
+        )
+      },
+      {
+        name: 'Delete Teacher Account - with students',
+        description: 'Warning message for teacher account deletion.',
+        story: () => (
+          <DeleteAccountDialog
+            {...DEFAULT_PROPS}
+            isTeacher={true}
+            warnAboutDeletingStudents={true}
+          />
+        )
+      },
+      {
+        name: 'Delete Teacher Account with students - 1 checkbox visible',
+        description: `
+        A teacher with students is only required to see/check
+        the first checkbox to delete their account.
+        `,
+        story: () => (
+          <DeleteAccountDialog
+            {...DEFAULT_PROPS}
+            isTeacher={true}
+            warnAboutDeletingStudents={true}
+            checkboxes = {
```

Comment:
formatting nit: make this a more compact one-liner as `checkboxes={getCheckboxes(false, true)}`

Reply:
(same on line 89)